### DISCUSSION TITLE
Sentry v6

### DISCUSSION THREAD

**Discussion by ueman** [getsentry/sentry-dart#419] [Ideas]
This is a discussion to keep track of feature and bug requests which should be in the next major version 6.

See the [Changelog](https://github.com/getsentry/sentry-dart/blob/main/CHANGELOG.md) for a list of changes.

**Comment by marandaneto** [comment]
if they are already issues, let's also add a label 6.0.0 like this one https://github.com/getsentry/sentry-dart/issues/405

**Comment by ueman** [comment]
# ⚠️ Possible breaking changes
- https://github.com/getsentry/sentry-dart/pull/391 could lead to breaking changes
    - Do we want to change the [`Transport`](https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/transport/transport.dart) interface or not? 
- https://github.com/getsentry/sentry-dart/pull/413 could lead to breaking changes
    -  Do we want to change the logging mechanism and name? See https://github.com/getsentry/sentry-dart/pull/413#discussion_r612528738
- https://github.com/getsentry/sentry-dart/issues/405 leads to a breaking change
    - Removal of `Sentry.currentHub` is a breaking change

**Comment by ueman** [comment]
I'd like to add an error/exception and stacktrace parameter to our logger.
See https://github.com/getsentry/sentry-dart/pull/453/files#discussion_r625680627
